app/serializers/tool_group_serializer.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class ToolGroupSerializer < ActiveModel::Serializer
+  attributes :id, :name
+  attribute :suggestions_weight, key: "suggestions-weight"
+
+  type "tool-group"
+
+  has_many :rule_languages, key: "rules-language"
+  has_many :rule_countries, key: "rules-country"
+  has_many :rule_praxes, key: "rules-praxis"
+  has_many :resource_tool_groups, key: "resource-tool-groups"
```

Comment:
use a key of `tools` for this

```suggestion
  has_many :resource_tool_groups, key: "tools"
```

Reply:
@cesarediaz there are still a few outstanding change requests on the tool-groups APIs

Reply:
solved